lua/telescope/builtin/files.lua

```diff
@@ -19,6 +19,8 @@ files.live_grep = function(opts)
         return nil
       end
 
+      if string.match(prompt, "[\\|%(%)]") then prompt = "" end
```

Comment:
This works on `live_grep` and still match the `\` or `()`, but if I do this on `grep_string`, it won't match `\` nor `()`.